[2022-07-19 10:08:09,925] {taskinstance.py:1159} INFO - Dependencies all met for <TaskInstance: simple_dag.respond scheduled__2022-07-19T10:07:00+00:00 [queued]>
[2022-07-19 10:08:09,933] {taskinstance.py:1159} INFO - Dependencies all met for <TaskInstance: simple_dag.respond scheduled__2022-07-19T10:07:00+00:00 [queued]>
[2022-07-19 10:08:09,933] {taskinstance.py:1356} INFO - 
--------------------------------------------------------------------------------
[2022-07-19 10:08:09,934] {taskinstance.py:1357} INFO - Starting attempt 1 of 1
[2022-07-19 10:08:09,934] {taskinstance.py:1358} INFO - 
--------------------------------------------------------------------------------
[2022-07-19 10:08:09,952] {taskinstance.py:1377} INFO - Executing <Task(PythonOperator): respond> on 2022-07-19 10:07:00+00:00
[2022-07-19 10:08:09,957] {standard_task_runner.py:52} INFO - Started process 6846 to run task
[2022-07-19 10:08:09,961] {standard_task_runner.py:79} INFO - Running: ['***', 'tasks', 'run', 'simple_dag', 'respond', 'scheduled__2022-07-19T10:07:00+00:00', '--job-id', '343', '--raw', '--subdir', 'DAGS_FOLDER/first_dag.py', '--cfg-path', '/tmp/tmpr5gyidco', '--error-file', '/tmp/tmpfcq6adqq']
[2022-07-19 10:08:09,963] {standard_task_runner.py:80} INFO - Job 343: Subtask respond
[2022-07-19 10:08:10,025] {task_command.py:370} INFO - Running <TaskInstance: simple_dag.respond scheduled__2022-07-19T10:07:00+00:00 [running]> on host 02ffb48252da
[2022-07-19 10:08:10,105] {taskinstance.py:1571} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=simple_dag
AIRFLOW_CTX_TASK_ID=respond
AIRFLOW_CTX_EXECUTION_DATE=2022-07-19T10:07:00+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2022-07-19T10:07:00+00:00
[2022-07-19 10:08:16,095] {logging_mixin.py:115} INFO - (2724, 1)
[2022-07-19 10:08:16,100] {python.py:173} INFO - Done. Returned value was: Greet Responded Again
[2022-07-19 10:08:16,127] {taskinstance.py:1400} INFO - Marking task as SUCCESS. dag_id=simple_dag, task_id=respond, execution_date=20220719T100700, start_date=20220719T100809, end_date=20220719T100816
[2022-07-19 10:08:16,179] {local_task_job.py:156} INFO - Task exited with return code 0
[2022-07-19 10:08:16,205] {local_task_job.py:273} INFO - 0 downstream tasks scheduled from follow-on schedule check
